Lib/test/test_urllib.py

```diff
@@ -712,7 +712,7 @@ def _reporthook(par1, par2, par3):
 
         with self.assertRaises(urllib.error.ContentTooShortError):
             try:
-                urllib.request.urlretrieve('http://example.com/',
+                urllib.request.urlretrieve(f'{support.TEST_HTTP_URL}/',
```

Comment:
What's the point of adding a slash? Why not using TEST_HTTP_URL directly?
```suggestion
                urllib.request.urlretrieve(support.TEST_HTTP_URL,
```